### ISSUE TITLE
Automatically alphabetize which-key menu?

### ISSUE THREAD

**Issue by macintacos** [VSpaceCode/vscode-which-key#6]
If you use the default configuration and create some new menus of your own, they appear at the bottom of the list. It'd be nice if they showed up where they'd be in alphabetical order (regardless of capitalization).

**Comment by stevenguh** [comment]
The original ideal is so that people can specify their own ordering. I think we can have an option in settings so the menu will be sorted alphabetically.

**Comment by macintacos** [comment]
Fair, but if we use the default modification, I can't figure out a way to reorder the existing menu. Everything I add to my settings gets put to the bottom of the menu.

**Comment by stevenguh** [comment]
If you meant the `bindingOverrides`, you should able to specifies the parameter of `position` like
``` json
{
  "whichkey.bindingOverrides": [
    {
      "keys": "g.s",
      "name": "Go to line",
      "type": "command",
      "command":"workbench.action.gotoLine",
      "position": 0
    },
    {
      "keys": "g.c",
      "name": "Go to line2",
      "type": "command",
      "command":"workbench.action.gotoLine",
      "position": 0
    }
  ]
}
```
The above example, will first remove any key that's associated to `<spc> g s` and insert the specified binding at position 0. The overrides are processed one by one, so in the case the resulting order of the menu in `<spc> g` will be
```
Go to line2
Go to line
...
```

Reorder like this can be very clunky, because it was meant to be surgically add and remove small number of default bindings. I will implement the sorting alphabetically soon.

**Comment by MarcoIeni** [comment]
maybe an option could be added. So you can explicitly add this to your settings.

**Comment by stevenguh** [comment]
Agree :) will work on it

**Comment by stevenguh** [comment]
Released `v0.7.5` that includes this option

**Comment by macintacos** [comment]
Quick turnaround, thanks a bunch!
